OpenLaTeXStudio/Editor/pom.xml

```diff
@@ -50,6 +50,7 @@
         <dependency>
             <groupId>org.netbeans.api</groupId>
             <artifactId>org-openide-awt</artifactId>
+            <version>${netbeans.version}</version>
```

Comment:
The versions should be declared in the parent pom.xml in the dependency management section